wagtail/wagtailcore/tests/test_page_model.py

```diff
@@ -67,13 +67,6 @@ def test_slug_is_autogenerated(self):
         retrieved_page = Page.objects.get(id=events_page.id)
         self.assertEqual(retrieved_page.slug, 'events-2')
 
-    def test_slug_must_be_unique_within_parent(self):
```

Comment:
I don't think so. A ValidationError can no longer occur since we always generate a unique slug.